drupal_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -x
+set -e
+
+REPO=`pwd`
+cd /tmp
+rm -rf drupal
+composer create-project --no-interaction drupal-composer/drupal-project:8.x-dev drupal
+cd drupal
+(cd vendor/twig && rm -rf twig && ln -sf $REPO twig)
+echo '$config["system.logging"]["error_level"] = "verbose";' >> web/sites/default/settings.php
```

Comment:
should we have some `after_failure` step in the Travis job which would display the latest logs ?

Reply:
This line allows to get detailed errors in the browser, which helps debugging errors locally.

Reply:
but is there a way to get the error message on Travis ?

Reply:
Not really. My idea (for now at least) is to have a red feedback when we break something and be able to replicate the error locally for easier debugging. The current script works well on my laptop, which is enough I think for now.